unix/vncserver/vncsession.c

```diff
@@ -503,6 +503,14 @@ run_script(const char *username, const char *display, char **envp)
     _exit(EX_OSERR);
 }
 
+static void
+usage(void)
+{
+    fprintf(stderr, "Syntax:\n");
+    fprintf(stderr, "    vncsession [-D] <username> <display>\n");
+    _exit(EX_USAGE);
```

Comment:
A normal `exit()` is fine here. The other call is because we're around `fork()` and `execvp()` which complicate things.

Reply:
Fixed in updated PR.